modules/twinkleprotect.js

```diff
@@ -1021,6 +1068,14 @@ Twinkle.protect.callback.evaluate = function twinkleprotectCallbackEvaluate(e) {
 		case 'protect':
 			// protect the page
 
+			if (form.editlevel.value === 'extendedconfirmed') {
+				if (!confirm('Extended confirmed protection is only for Arbitration enforcement. ' +
+						'Please confirm that the subject of this page is a qualifying topic (see WP:30/500)'
```

Comment:
Full stop needed at the end of the message. And you can use && to merge the two if statements into one and avoid crazily deep indent levels.